src/components/InventoryHostStaleness/constants.js

```diff
@@ -0,0 +1,452 @@
+    ) {
+      setIsFormValid(false);
+      return (
+        <p className="pf-u-font-size-sm pf-v5-u-danger-color-100">
+          Culling must be after stale warning
+        </p>
+      );
+    } else {
+      setIsFormValid(true);
+    }
+  }
+};
+export const HostStalenessResetDefaultPopover = () => {
+  return (
+    <Popover
+      aria-label="Inventory Groups popover"
+      headerContent={
+        <Title headingLevel="h4">Orginization level setting</Title>
+      }
+      position="top"
+      bodyContent={
+        <Flex
+          direction={{ default: 'column' }}
+          spaceItems={{ default: 'spaceItemsNone' }}
+        >
+          <FlexItem>
+            {' '}
+            <p className="pf-u-font-size-sm">
+              - Systems are marked as stale after 1 day since last check-in.
+            </p>
+          </FlexItem>
+          <FlexItem>
+            {' '}
+            <p className="pf-u-font-size-sm">
+              - Systems are marked as stale warning after 14 days since last
+              check-in.
+            </p>
+          </FlexItem>
+
+          <FlexItem>
+            {' '}
+            <p className="pf-u-font-size-sm">
+              - Systems are culled after 30 days since last check-in.
+            </p>
+          </FlexItem>
+        </Flex>
+      }
+    >
+      <Button
+        variant="plain"
+        aria-label="Open Inventory groups popover"
+        style={{ padding: 0 }}
+      >
+        <OutlinedQuestionCircleIcon className="pf-u-ml-md" />
+      </Button>
+    </Popover>
+  );
+};
+
+export const InventoryHostStalenessPopover = () => {
+  return (
+    <Popover
+      aria-label="Inventory Groups popover"
+      headerContent={
+        <Title headingLevel="h4">Orginization level setting</Title>
+      }
+      hasAutoWidth
+      position="top"
+      bodyContent={
+        <div>
+          <p className="pf-u-font-size-sm">
+            Configure the number of days it will take for your systems to be
+            marked as stale, stale warning, and be cullled.
+          </p>
+          <Flex
+            direction={{ default: 'column' }}
+            spaceItems={{ default: 'spaceItemsNone' }}
+          >
+            <FlexItem>
+              <p className="pf-u-font-size-sm">By default:</p>
+            </FlexItem>
+            <FlexItem>
+              <p>
+                - Systems are marked as stale after 1 day since last check-in.
+              </p>
+            </FlexItem>
+            <FlexItem>
+              {' '}
+              <p className="pf-u-font-size-sm">
+                - Systems are marked as stale warning after 14 days since last
+                check-in.
+              </p>{' '}
+            </FlexItem>
+            <FlexItem>
+              {' '}
+              <p className="pf-u-font-size-sm">
+                - Systems are culled after 30 days since last check-in.
+              </p>{' '}
+            </FlexItem>
+          </Flex>
```

Comment:
I think we are missing "Learn more" link in the very bottom of the popover content

Reply:
We removed this, the mocks just havent been updated to show it by UX told me to remove it for the time being.